src/stories/media.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs/react';
+import styles from "@sambego/storybook-styles";
+import { withInfo } from '@storybook/addon-info';
+
+import VideoPlayer from '../components/video_player';
+
+const stories = storiesOf('Media', module);
+
+stories.addDecorator(withKnobs)
+    .addDecorator(styles({
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        height: '100vh',
+    }));
+
+stories.add('Video player', withInfo(
+    'A Video player'
+)(() =>
+    <VideoPlayer width={text('width', '')} height={text('height', '')} url={text('url', 'https://www.youtube.com/watch?v=160yAufQ-is')} />
```

Comment:
Can you please make the width and height labels start with a capital

Reply:
done